src/apis/user/getUser.ts

```diff
@@ -0,0 +1,13 @@
+import { axiosInstance } from "@apis/axiosInstance";
+
+import { GetUserResponse } from "@type/apis/users/GetUser";
+
+import { DOMAIN } from "@constants/api";
+
+export const getUser = async (userId: string) => {
+  const { data } = await axiosInstance.get<GetUserResponse>(
+    DOMAIN.GET_USER(userId)
+  );
+
+  return data;
+};
```

Comment:
이 api의 경우 토큰이 필요하지 않습니다! useAuth 속성을 false로 해주세요.

```suggestion
export const getUser = async (userId: string) => {
  const { data } = await axiosInstance.get<GetUserResponse>(
    DOMAIN.GET_USER(userId),
    {
      useAuth: false
    }
  );

  return data;
};
```